js/components/graph/Graph.js

```diff
@@ -616,6 +630,7 @@ export default class Graph extends React.Component {
       this.state.verticalPanFactor === 0;
 
     // Mouse events for draw tool
+    // onDraw defaults to false
```

Comment:
I don't think you need this comment, this part does work as intended even if the functions are not currently being used.